content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,27 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Delegation of managing a Job object to external controller
+
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
```

Comment:
We should mention that feature gate should be enabled to use this feature.

Can we also mention the risks of disabling the feature gate or downgrading to pre-alpha version?

Reply:
Sure, I will update it.

Reply:
Updated, PTAL